setup.py

```diff
@@ -72,6 +72,14 @@
         ],
         "vw": [
             "vowpalwabbit",
+        ],
+        "nlp": [
+            "ray[tune]>=1.2.0",
+            "transformers",
+            "wandb==0.10.26",
+            "datasets==1.4",
```

Comment:
too specific?

Reply:
I remember 1.4 is required or there's an error